src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/DirectoryExistsTests.cs

```diff
@@ -49,16 +49,38 @@ public void DirectoryExists_Closed_ThrowsInvalidOperationException()
             }
         }
 
-        [ActiveIssue(25665)]
         [Fact]
-        public void DirectoryExists_RaisesArgumentException()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void DirectoryExists_RaisesArgumentException_Desktop()
         {
             using (IsolatedStorageFile isf = IsolatedStorageFile.GetUserStoreForAssembly())
             {
                 AssertExtensions.Throws<ArgumentException>("path", null, () => isf.DirectoryExists("\0bad"));
             }
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void DirectoryExists__Invalid_Path_Core_Windows()
+        {
+            using (IsolatedStorageFile isf = IsolatedStorageFile.GetUserStoreForAssembly())
+            {
+                Assert.False(isf.DirectoryExists("\0bad"));
+            }
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public void DirectoryExists__InvalidPath_Core_Unix()
+        {
+            using (IsolatedStorageFile isf = IsolatedStorageFile.GetUserStoreForAssembly())
+            {
+                Assert.True(isf.DirectoryExists("\0bad"));
```

Comment:
Why does this True differ from the Windows' False?

Reply:
"\0bad" is a valid name for directory in unix but that is not the case with windows

Reply:
Then why is the name of this test "InvalidPath"?

Reply:
When we try to create the path using .dotnet core app on linux, it throws invalid character error if the name contains the invalid character("/").  That's why it was earlier named Invalid_path. 
Now after removing the check , it no longer checks at the string name level and proceeds to create the file.
I have changed name to  DirectoryExists__Special_Core_Unix()